src/Table/TableBody.js

```diff
@@ -204,7 +205,7 @@ class TableBody extends Component {
       <TableRowColumn
         key={key}
         columnNumber={0}
-        style={{width: 24}}
+        style={Object.assign({width: 24}, isDisabled && {cursor: 'not-allowed'})}
```

Comment:
Why not a ternary inside the object literal for the `cursor` property instead of an object.assign() with a short circuit evaluation + new object literal?